### DISCUSSION TITLE
getting the correct (sequential) docker tag

### DISCUSSION THREAD

**Discussion by Mavrek-Nine-Two** [renovatebot/renovate#16568] [Request Help]
Good day.
I am trying to use renovatebot to handle just dockerfiles, with my first mvp a single-liner project I am trying to update with the bot.

i.e:
`FROM:myrepository/image:1.5.1`

I have two streams of image in my repository, and sometimes the same image will be tagged against both.
Stream 1 tags are like :  `1.2.3`
Stream 2 tags are like : `1234.5678.9123`

On execution the bot  does "its' thing" but the results are not what I need. 
In this example, the image tag "1.5.1" has been superceded by "1.5.6" and a stream2 tag exists as well.
When  I run the bot, instead of seeing it grab the 1.5.6 I see it grabs the stream2 value, ie:
1.5.1 -> 1232.123.123.123

Per the documentation,  what I should be seeing is the behaviour I need - which would be reflected as;  1.5.1 -> 1.5.6  ,but this is NOT what is happening.   

The rule I have shouldn't even match on the major, but it does seem to be doing so.. 
So - how do I achieve the (correct!) desired behaviour? 

--
My renovate configuration is bare-bones (only specifies the dockerfile manager), and the renovate.json in the mvp uses only the single directive:

```
"packageRules": [
        {
          "updateTypes": ["minor","patch"],
          "automerge": true
        }
      ]
```

Thanks!

**Comment by viceice** [comment]
major updates are enabled by default, so renovate will open a PR for it.

**Comment by PhilipAbed** [comment]
try this:
```
"packageRules": [
    {
      "matchUpdateTypes": ["major"],
      "dependencyDashboardApproval": true
    }
  ]
```

and `major` updates will only appear in dashboard instead of opening a PR

The rule you have is for `minor` + `patch` and it says `Automerge=true`,

that means they will be automerged automatically , but `major` updates will still raise a PR but won't merge it automatically
